src/components/App.js

```diff
@@ -98,6 +100,24 @@ const App = ({
         setPreviousLocation(location.pathname)
     }
 
+    const onResponseReceived = response => {
+        const metadata = Object.entries(response.metaData.items).reduce(
+            (obj, [id, item]) => {
+                obj[id] = {
+                    id,
+                    name: item.name || item.displayName,
+                    displayName: item.displayName,
+                    dimensionItemType: item.dimensionItemType, // TODO needed?
```

Comment:
needed? 🤔

Reply:
I don't know, for now we don't use the metadata in the store, because the `DataTable` is rendered in the same component where the analytics response is fetched, so the metadata is read directly from there.
I didn't check how we use that in DV, but perhaps somewhere in the Layout is used?